test/EndToEnd/tests/PackageRestoreTest.ps1

```diff
@@ -371,6 +371,9 @@ function RemoveDirectory {
     {
         if (Test-Path $dir)
         {
+            # because -Recurse parameter in Remove-Item has a known issue so using Get-ChildItem to
+            # first delete all the children and then delete the folder.
+            Get-ChildItem $dir -Recurse | Remove-Item -Force -Recurse -ErrorAction SilentlyContinue
             Remove-Item -Recurse -Force $dir -ErrorAction SilentlyContinue
```

Comment:
remove the second remove-item

Reply:
No its required to delete the folder itself. First line will remove all the content except the folder itself